code/frameworks/angular/package.json

```diff
@@ -48,16 +84,12 @@
     "!src/**/*"
   ],
   "scripts": {
-    "check": "yarn run -T tsc",
-    "prep": "rimraf dist && jiti ../../../scripts/prepare/tsc.ts"
+    "check": "jiti ../../../scripts/prepare/check.ts",
+    "prep": "jiti ../../../scripts/prepare/addon-bundle.ts"
```

Comment:
@ndelangen why use `addon-bundle.ts` instead of `bundle.ts` like the other frameworks are using?

Reply:
with `addon-bundle` you can be more explicit about which entrypoints have which runtime target (browser,node).